src/main/lrsql/system/openapi.clj

```diff
@@ -0,0 +1,12 @@
+(ns lrsql.system.openapi
```

Comment:
Note the other `system` namespaces are all for `component`s. It's a little counter-intuitive, but in the convention of the project stuff like this would go under an `init` ns

Reply:
moved

Reply:
where to?

Reply:
`lrsql/admin/interceptors/openapi.clj`